e2e/helpers/wallet.helper.ts

```diff
@@ -53,12 +65,18 @@ export class WalletHelper {
         return this.bitcoinRPCUtil.mineToAddress(numOfBlocks, walletAddress);
     }
 
-    async addFundToUTXO(payment: Payment, amount): Promise<UTXO> {
+    async addFundToUTXO(
+        payment: Payment,
+        amount: number,
+        addressType: AddressType,
+        index: number,
```

Comment:
you are passing this to the function and just returning it. If you don't need to use it in the function then why pass it as an argument?

Reply:
this is used for 3 main reasons:
1. providing complete UTXO information
2. maintaining context for downstream processes
3. consistency and clarity